docs/cli.md

```diff
@@ -51,7 +51,7 @@ Or you can use compile and install the plugin from your `$GOPATH/src/github.com/
 
 | Syntax                                     | Description                                                                                   |
 | ------------------------------------------ | --------------------------------------------------------------------------------------------- |
-| `kubectl kudo install <name> [flags]`      | Install a Framework from the official [KUDO repo](https://github.com/kudobuilder/frameworks). |
+| `kubectl kudo install <name> [flags]`      | Install a Operator from the official [KUDO repo](https://github.com/kudobuilder/operators). |
```

Comment:
I assume we are going to make this change

Reply:
ya. is the URL updated in the repo codebase as well for this?